schema/playground/launcher.schema

```diff
@@ -11,6 +11,7 @@ type LaunchCommandRequest {
     improbable.Vector3f impact_point = 2;
     improbable.Vector3f launch_direction = 3;
     float launch_energy = 4;
+    int64 launcher = 5;
```

Comment:
What does this represent?

Reply:
in that case, could we make it more obvious? :)

Reply:
Next commit is doing mostly renaming :)